LayoutTests/webgl/webgl-draft-extensions-flag-default-expected.txt

```diff
@@ -0,0 +1,15 @@
+This test outputs which WebGL draft extensions are available.
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+TEST COMPLETE: 5 PASS, 0 FAIL
+
+
+PASS webgl2:WEBGL_draw_instanced_base_vertex_base_instance: Supported
+PASS webgl2:WEBGL_draw_instanced_base_vertex_base_instance: Has object.
+PASS webgl2:WEBGL_multi_draw_instanced_base_vertex_base_instance: Supported
```

Comment:
Since WebKit's test expectations require exact matches, any hardware on the CI that can't support all draft extensions won't be able to pass this test. Not sure if anything is to be done about this; I agree it's better to be clearer in the test output rather than just printing the extension name and "passed", and only printing the failure modes if the extension wasn't supported.

Reply:
Yeah, it'll be a problem when there's a draft extension that is per-platform-per-gpu variant test. I think we should alter that then.
The idea of the test bunch is to try to have some coverage in layout test to test whether or not the draft extensions are shipping by mistake or not.